### DISCUSSION TITLE
override calibration Libre 2

### DISCUSSION THREAD

**Discussion by muccioli** [NightscoutFoundation/xDrip#3542] [General]
Good morning.
 3 days ago, I started using libre 2 plus, together with juggluco to send readings to xdrip, I had some problems like disconnection and some others, which I'm solving, but I don't know if any modifications were made and I can't find out anymore. Right, it's the following:
The "override calibration" option no longer appears in the xdrip menu. At times, only “initial calibration” appears. How to make the "override calibration" option work again? Or is this normal with the Libre 2 Plus?
xdrip version xDrip-plus-20240611-1958827
Juggluco version Juggluco-8.0.24-arm64
Phone Samsung A71
Android 13
![IMG_20240616_100306505](https://github.com/NightscoutFoundation/xDrip/assets/118318032/dc23924f-68dc-416c-a974-19acd391896f)

Could anyone help?
Thanks

**Comment by Navid200** [comment]
I don't think you can override a calibration that has not been performed by xDrip.

If you are using xDrip as a follower of another app, it cannot be xDrip that would perform any calibrations.

Anyway, you can install an older version to verify if there has been a change.

**Reply by PhilTU** [reply]
xDrip+ can handle the L:ibre 2+ sensor itself since last week. Here is nightly version link https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.06.11 

updated OOP2 to go with it. https://xdrip.readthedocs.io/en/latest/use/misc/#oop2 https://drive.google.com/file/d/15W6L3w3dwCNzESoBO2mjfsWmZafw9s73/view 

setup instructions https://xdrip.readthedocs.io/en/latest/install/libre2/ we recommend starting with no calibration

**Reply by muccioli** [reply]
So, the version of xdrip I use is the same as what you are suggesting.

As I said, I used juggluco, I couldn't make the settings with oop2.

Thanks

Navid200, With a previous, stable version, 04/19/24, with the Libre 1, there was this override calibration option.

Thanks

**Reply by PhilTU** [reply]
> I couldn't make the settings with oop2.

did you have the updated OOP2 as well ?

**Reply by muccioli** [reply]
Yes, i use OOP2 version oop2_c6_09_31

**Reply by PhilTU** [reply]
ok, what region ?

**Reply by muccioli** [reply]
Brazil.

**Reply by PhilTU** [reply]
ok, might be a different sensor variant like the Russian one.

**Reply by muccioli** [reply]
Thanks.

**Comment by PhilTU** [comment]
I don't recall an override calibration. If the hardware data source is Libre 2 patched you can change to "non fixed slopes" or similar.

https://xdrip.readthedocs.io/en/latest/calibrate/troubleshoot/?h=calibration explains some ways to resolve calibration problems

**Reply by muccioli** [reply]
"non fixed slopes" Is it an xdrip configuration option?

**Reply by muccioli** [reply]
Now I tried to stop the sensor and clear the calibrations, then no message to calibrate appeared, I can't recalibrate.

**Reply by PhilTU** [reply]
so no calibration option on main menu ? If not you have the possiblity to use treatment valuies for calibration 
In xDrip+ settings / other settings / less common settings / Advanced calibration under "Use treatment BG values" you can choose auto, ask each time, never or always. So when you enter fingerprick value as treatment it can be used for "calibration".

non fixed slopes is in same place scroll down near bottom

**Reply by muccioli** [reply]
Many thanks, i try this option.

**Reply by muccioli** [reply]
After several configuration attempts, I redid the initial calibration and now the Add-Calibration option appears.
Thank you very much for the tips. I hope it stays stable and working.
Regards

**Reply by Navid200** [reply]
The add calibration option does not appear after a sensor start until there have been three readings.

**Reply by muccioli** [reply]
Ok. Thanks.
